packages/studioCMS/src/integrations/studioCMSDashboard/routes/dashboard/pages/new/page.astro

```diff
@@ -106,93 +145,109 @@ const pageTitle = `New Page | ${contextConfig.title}`;
         </sl-alert>
         )}
     </div>
-    <div class="pb-12 mt-5">
-
-        <div class="text-5xl font-bold">Create New Page</div>
-
-        <div class="py-2">
-
-            <center>
-
-                <form method="post" enctype="multipart/form-data" >
-                    <label for="title" class="text-lg">Page Title</label><br>
-                    <input 
-                        type="text" 
-                        id="title" 
-                        name="title" 
-                        class="border-solid border-2 border-gray-300 rounded-lg p-2 w-96"
-                        placeholder="Your Page Title" 
-                        required><br>
-                    <label for="slug" class="text-lg">Slug</label><br>
-                    <input 
-                        type="text" 
-                        id="slug" 
-                        name="slug" 
-                        class="border-solid border-2 border-gray-300 rounded-lg p-2 w-96"
+    <div class="container">
```

Comment:
tbh with ya, I'd like new and edit to just be the same component but just given different default values. So thatthe view, new and edit psges dont repeat the same stuff

Reply:
There will be a few changes and updates with the planned API endpoint system (including trying to componentize as many of the pages that are duplicating info as possible! (and moving to astrolace now that i have that working)